client/app/pages/queries/QueryView.jsx

```diff
@@ -125,51 +142,47 @@ function QueryView(props) {
                 onChangeTab={setSelectedVisualization}
                 onAddVisualization={addVisualization}
                 onDeleteVisualization={deleteVisualization}
+                cardStyle
```

Comment:
Let's not make this configurable but use this on both pages, to make sure it makes sense in both pages (as it has to be the same).